### DISCUSSION TITLE
Cursor thickness

### DISCUSSION THREAD

**Discussion by MMVS-79** [coteditor/CotEditor#1663] [Q&A] [answered]
Is there any way to change the cursor thickness to be more thick like this (example of my sublime text cursor):
<img width="177" alt="image" src="https://github.com/coteditor/CotEditor/assets/174728384/2e022220-5ccd-43e8-8379-35c74f8cfbff">

**Comment by MMVS-79** [comment]
Also, can I turn off the blinking?

**Comment by 1024jp** [comment, marked as answer]
The short answer is "No." See #1605 for the further interpretation.
